### PR TITLE
INC-615  Fix Incentives API database connection issues

### PR DESCRIPTION
**Pull request by Mjwillis**
Signed-off-by: mjwillis <michael.willis@digital.justice.gov.uk>

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -25,12 +25,12 @@ dependencies {
...
+  implementation("org.jetbrains.kotlinx:kotlinx-coroutines-reactor:1.6.3")
 
-  implementation("org.flywaydb:flyway-core:8.5.12")
+  implementation("org.flywaydb:flyway-core:8.5.13")
   implementation("com.vladmihalcea:hibernate-types-52:2.16.2")
-  runtimeOnly("io.r2dbc:r2dbc-postgresql")
+  runtimeOnly("org.postgresql:r2dbc-postgresql:0.9.1.RELEASE")
   runtimeOnly("org.springframework.boot:spring-boot-starter-jdbc")
   runtimeOnly("org.postgresql:postgresql:42.4.0")
   implementation("com.zaxxer:HikariCP:5.0.1")
```
Comment (petergphillips) [inline build.gradle.kts:36]: don't need hikari cp

File: build.gradle.kts [hunk 2]
```diff
@@ -25,12 +25,12 @@ dependencies {
...
   implementation("uk.gov.justice.service.hmpps:hmpps-sqs-spring-boot-starter:1.1.3")
 
-  implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.2")
-  implementation("org.jetbrains.kotlinx:kotlinx-coroutines-reactor:1.6.2")
+  implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.3")
+  implementation("org.jetbrains.kotlinx:kotlinx-coroutines-reactor:1.6.3")
 
-  implementation("org.flywaydb:flyway-core:8.5.12")
+  implementation("org.flywaydb:flyway-core:8.5.13")
   implementation("com.vladmihalcea:hibernate-types-52:2.16.2")
```
Comment (petergphillips) [inline build.gradle.kts:32]: found this hibernate-types thing in our code base too - not sure what it is doing there

**Review by petergphillips — APPROVED** [review]
LGTM

**Review by petergphillips — DISMISSED** [review]
LGTM

**Review by mikehalmamoj — APPROVED** [review]

**Pull request merged**